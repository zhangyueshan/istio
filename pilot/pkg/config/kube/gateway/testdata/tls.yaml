apiVersion: networking.x-k8s.io/v1alpha1
kind: GatewayClass
metadata:
  name: istio
spec:
  controller: istio.io/gateway-controller
---
apiVersion: networking.x-k8s.io/v1alpha1
kind: Gateway
metadata:
  name: gateway
  namespace: istio-system
spec:
  class: istio
  listeners:
    - hostname:
        match: Domain
        name: domain.example
      port: 80
      protocol: TLS
      tls:
        certificateRefs:
          - name: my-cert
        options: {}
        minimumVersion: TLS1_2
