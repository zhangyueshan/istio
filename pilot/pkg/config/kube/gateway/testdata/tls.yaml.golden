apiVersion: networking.istio.io/v1alpha3
kind: Gateway
metadata:
  creationTimestamp: null
  name: gateway-istio-autogenerated-k8s-gateway
  namespace: istio-system
spec:
  selector:
    istio: ingressgateway
  servers:
  - hosts:
    - '*.domain.example'
    port:
      name: tls-80-gateway-gateway-istio-system
      number: 80
      protocol: TLS
    tls:
      credentialName: my-cert
      minProtocolVersion: TLSV1_2
      mode: SIMPLE
---
